components/usage/cmd/run.go

```diff
@@ -72,6 +73,7 @@ func run() *cobra.Command {
 	}
 
 	cmd.Flags().BoolVar(&verbose, "verbose", false, "Toggle verbose logging (debug level)")
+	cmd.Flags().DurationVar(&schedule, "schedule", 1*time.Minute, "The schedule on which the reconciler should run")
```

Comment:
Side-note: Maybe also increase the default reconciliation loop period to 30min or 1h? (I'm a little worried about high-frequency logging noise)

Reply:
Rebased to change the default frequency to 1 hour.